docs/node/node_config.yaml

```
@@ -105,10 +105,28 @@ encryption:
 # Define who is allowed to run which algorithms on this node.
 policies:
   # Control which algorithm images are allowed to run on this node. This is
-  # expected to be a valid regular expression.
+  # expected to be a valid regular expression. If you don't specify this, all algorithm
+  # images are allowed to run on this node (unless other policies restrict this).
   allowed_algorithms:
     - ^harbor2\.vantage6\.ai/[a-zA-Z]+/[a-zA-Z]+
     - ^myalgorithm\.ai/some-algorithm
+
+  # It is also possible to allow all algorithms from particular algorithm stores. Set
+  # these stores here. They may be strings or regular expressions. If you don't specify
+  # this, algorithms from any store are allowed (unless other policies prevent this).
+  allowed_algorithm_stores:
+    # allow all algorithms from the vantage6 community store
+    - https://store.cotopaxi.vantage6.ai
+    # allow any store that is a subdomain of vantage6.ai
+    - ^https://*\.vantage6\.ai$
```

Heh these regexes are error prone I think

```suggestion
    - ^https://.*\.vantage6\.ai$
```

Or maybe even more precise:
```suggestion
    - ^https://[a-z]+\.vantage6\.ai$
```

---

Good catch, I really suck at regex :-) I've added the last version.